src/plugins/intel_cpu/src/dnnl_postops_composer_legacy.cpp

```diff
@@ -95,6 +95,24 @@ void DnnlPostOpsComposerLegacy::appendBinary(const dnnl::algorithm alg, const st
     args[DNNL_ARG_ATTR_MULTIPLE_POST_OP(ops.len() - 1) | DNNL_ARG_SRC_1] = mem;
 }
 
+void DnnlPostOpsComposerLegacy::appendPReLU(const std::vector<float>& data) {
```

Comment:
May I know what is the difference between class DnnlPostOpsComposerLegacy and DnnlPostOpsCompose. I can check only fc use DnnlPostOpsCompose. Do we have plan to remove DnnlPostOpsComposeLegacy for long term or will keep 2 both in the code.